packages/components/src/horizontal-bar-list/stats-card.scss

```diff
@@ -136,6 +149,7 @@
 		flex-direction: column;
 		justify-content: space-between;
 		height: 100%;
+		position: relative;
```

Comment:
The `position` attribute was added here as it didn't affect the other cards. If preferable, it can be moved to the `.stats-card__content--hero` class for better concern separation.